doc_templates/README.md

```diff
@@ -0,0 +1,28 @@
+# CLI documentation templates
+
+These templates provide extra information for the automatically generated CLI
+documentation.
+
+## Adding new templates
+
+By default, no template is necessary. If you want to modify the automatically
+generated CLI documentation, you can lookup the filename for the specific
+command in `../docs/command_line_tool/` and add a file with identical name to
+this repository to use as a template.
```

Comment:
```suggestion
this directory to use as a template.
```

Reply:
Prepositional phrases suck... But it might be clearer if we re-order them?

> add a file to this directory with an identical name for use as a template

Reply:
It might be because I'm not a native speaker, but I honestly like @maxrake's suggestion much better (though maybe that's just because it's what I intended to write?).

Reply:
I am a native speaker and I have no clue which is actually better...

Using @maxrake's suggestion is fine.